clear-containers-image/release_image_github.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+script_name="${0##*/}"
+script_dir=$(dirname $(realpath -s "$0"))
+
+tmp_dir=$(mktemp -t -d cc-bump.XXXX)
+release_tool="${tmp_dir}/release-tool"
+osbuilder_dir="${tmp_dir}/osbuilder/"
+#FIXME: change to clearcontainers after is merged https://github.com/clearcontainers/osbuilder/pull/48
+repo_owner="clearcontainers"
+repo_owner="jcvenegas"
+push=false
+
+versions_file="https://raw.githubusercontent.com/clearcontainers/runtime/master/versions.txt"
+os_version="$(curl -Ls ${versions_file} | grep '^clear_vm_image_version=' | cut -d= -f 2)"
+agent_version="$(curl -Ls ${versions_file} | grep '^cc_agent_version=' | cut -d= -f 2)"
+#Fixme: remove after https://github.com/clearcontainers/runtime/pull/923 is merged
```

Comment:
I've just merged this change, so can you remove this comment and adjust `agent_version=` below?